crates/primitives/Cargo.toml

```diff
@@ -18,8 +18,8 @@ reth-codecs.workspace = true
 reth-ethereum-forks.workspace = true
 reth-static-file-types.workspace = true
 reth-trie-common.workspace = true
-reth-chainspec.workspace = true
 revm-primitives = { workspace = true, features = ["serde"] }
+reth-chainspec = { workspace = true, optional = true }
```

Comment:
why is this now optional?

Reply:
it was used in `block_coinbase` fn, but now only for optimism feature